docs/ml/deploy/_index.md

```diff
@@ -46,110 +39,44 @@ Follow [these instructions](/registry/advanced/mlmodel-design/) to design your m
-- a model trained outside the Viam platform that you have [uploaded](/ml/upload-model/)
+- a model from [the registry](https://app.viam.com/registry)
+- a model trained outside the Viam platform that you have uploaded
 - a model available on your machine
 
-{{< tabs >}}
-{{% tab name="Builder" %}}
-
-Navigate to your machine's **Config** tab on the [Viam app](https://app.viam.com/robots).
-Click the **Services** subtab and click **Create service** in the lower-left corner.
-Select the `ML Model` type, then select the `TFLite CPU` model.
-Enter a name for your service and click **Create**.
-
-You can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
-
-{{< tabs >}}
-{{% tab name="Deploy Model on Robot" %}}
-
-1. To configure your service and deploy a model onto your machine, select **Deploy Model On Robot** for the **Deployment** field.
-
-2. Click on **Models** to open a dropdown with all of the ML models available to you privately, as well as all of the ML models available in [the registry](https://app.viam.com), which are shared by users.
-   You can select from any of these models to deploy on your robot.
-
-{{<imgproc src="/services/deploy-model-menu.png" resize="700x" alt="Models dropdown menu with models from the registry.">}}
-
-{{% alert title="Tip" color="tip" %}}
-To see more details about a model, open its page in [the registry](https://app.viam.com).
-{{% /alert %}}
-
-3. Also, optionally select the **Number of threads**.
-
-{{<imgproc src="/services/deploy-model.png" resize="700x" alt="Create a machine learning models service with a model to be deployed">}}
+## Used with
 
-{{% /tab %}}
-{{% tab name="Path to Existing Model On Robot" %}}
+{{< cards >}}
+{{< relatedcard link="/ml/vision/">}}
+{{< relatedcard link="/components/board/">}}
+{{< relatedcard link="/components/camera/">}}
+{{< /cards >}}
 
-1. To configure your service with an existing model on the machine, select **Path to Existing Model On Robot** for the **Deployment** field.
-2. Then specify the absolute **Model Path** and any **Optional Settings** such as the absolute **Label Path** and the **Number of threads**.
+After deploying your model, you need to configure an additional service to use the deployed model.
+For example, you can configure an [`mlmodel` vision service](/ml/vision/) and a [`transform` camera](/components/camera/transform/) to visualize the predictions your model makes.
 
-![Create a machine learning models service with an existing model](/services/available-models.png)
+## Versioning for deployed models
 
-{{% /tab %}}
-{{< /tabs >}}
+If you upload or train a new version of a model, Viam automatically deploys the `latest` version of the model to the machine.
+If you do not want Viam to automatically deploy the `latest` version of the model, you can change the `packages` configuration in the [Raw JSON machine configuration](/build/configure/#the-config-tab).
```

Comment:
it's specifically the `packages` field value that gets configured when you add a model. @sguequierre I think showing the packages field with the array around it might help

```
"packages": [
    {
      "name": "redcar",
      "type": "ml_model",
      "version": "latest",
      "package": "ccb74b53-1235-4328-a4b9-91dff1915a50/redcar"
    },
    {
      "name": "redcar",
      "type": "ml_model",
      "version": "latest",
      "package": "ccb74b53-1235-4328-a4b9-91dff1915a50/redcar"
    }
  ],
```

Reply:
Showing it here again? I don't know if that's necessary but if you think it's important I can